README.md

```diff
@@ -58,14 +75,16 @@ await mongoSchedule.disconnect();
-| maxRunning    | `number`   | true     | `0`     | Maximum number of job executions that is allowed at a time. Set to 0 for no max. The schedule will trigger no more job executions if maxRunning is reached. However, there is no guarantee that the schedule always respects the limit; in rare cases with multiple Momo instances maxRunning may be exceeded. |
-| handler       | `function` | false    |         | The function to execute.                                                                                                                                                                                                                                                                                       |
+You can instantiate a momo job using the `MomoJobBuilder` class. It provides the following setter methods:
+
+| setter           | parameters                       | mandatory | default value | description                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| ---------------- |----------------------------------| --------- | ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| withName         | `string`                         | true      |               | The name of the job. Used as a unique identifier.                                                                                                                                                                                                                                                                                                                                                                                                            |
+| withSchedule     | `string`,`number` (default: `0`) | false     |               | Either this setter or `withCronSchedule()` must be called. Specifies the time interval at which the job is started. Time intervals in human-readable formats ( like '1 minute', 'ten days' or 'twenty-one days and 2 hours' ) are accepted. Check documentation of [ human-interval ](https://www.npmjs.com/package/human-interval) library for details.If the job never ran before, the job will run after `firstRunAfter` milliseconds for the first time. |
```

Comment:
there's a space too much: "( like"

Reply:
You can move the space here instead: "details.If" :D

Reply:
Lol. :D